doc/user_guide/psyke.rst

```diff
@@ -366,7 +366,7 @@ The generated code is now:
 
 Examples in ``examples/lfric/eg12`` directory demonstrate how to
```

Comment:
As mentioned earlier, my feeling is that we could wipe eg12 for the time being - leave it as a vacant slot for a new example? That's probably a separate PR though.

Reply:
I agree we should clean up and rationalise the examples, I also wanted to try including the README from the relevant folder and even maybe the main script in the webpage to avoid duplicating text, but I want to try this separately.